src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -163,7 +164,7 @@ const ChatItemMyTextV2 = ({
           </View>
         )}
         {attachments.length > 0 && <ChatItemAttachment attachments={attachments} />}
-        {attachments.length <= 0 && <Text style={styles.text}>{message}</Text>}
+        {attachments.length <= 0 && <LinkableText style={styles.text} text={message} />}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from a simple `Text` component to the new `LinkableText` component is good for handling clickable links within chat messages. However, it's important to ensure that the `LinkableText` component properly sanitizes the URLs to prevent any potential security risks such as XSS (Cross-Site Scripting) attacks. If not already implemented, consider using a library like DOMPurify to sanitize the URLs.
```diff
- {attachments.length <= 0 && <LinkableText style={styles.text} text={message} />}
+ {attachments.length <= 0 && <LinkableText style={styles.text} text={DOMPurify.sanitize(message)} />}
```
Please note that you need to import DOMPurify at the top of your file if you decide to use it.
```javascript
import DOMPurify from 'dompurify';
```
Also, remember to install it via npm or yarn.
```bash
npm install dompurify
# or
yarn add dompurify
```
This is just a suggestion and may require additional changes based on your project setup.


<!-- This is an auto-generated comment by OSS CodeRabbit -->